autokeras/blocks/basic.py

```diff
@@ -974,14 +976,14 @@ def from_config(cls, config):
     def build(self, hp, inputs=None):
         input_tensor = nest.flatten(inputs)[0]
 
-        tokenizer_layer = keras_layers.BertTokenizer(
-            max_sequence_length=utils.add_to_hp(self.max_sequence_length, hp)
+        preset_name = "bert_base_en_uncased"
+        tokenizer_layer = keras_nlp.models.BertPreprocessor.from_preset(
+            preset_name,
+            sequence_length=utils.add_to_hp(self.max_sequence_length, hp),
         )
-        output_node = tokenizer_layer(input_tensor)
-
-        bert_encoder = keras_layers.BertEncoder()
+        bert_encoder = keras_nlp.models.BertBackbone.from_preset(preset_name)
 
-        output_node = bert_encoder(output_node)
-        bert_encoder.load_pretrained_weights()
+        output_node = tokenizer_layer(tf.reshape(input_tensor, [-1]))
```

Comment:
Why do you need a reshape here? What is the shape of the input tensor?

Reply:
The shape is `[batch_size, 1]`. The input tensor is not `["This is the first sentence.", "This is the second one."]`, but `[["This is the first sentence."], ["This is the second one."]]`. It has one more dimension.